app/views/layouts/base.html.erb

```diff
@@ -9,11 +9,11 @@
     <%= yield(:meta) %>
 
     <%= favicon_link_tag "favicon.ico"%>
-
-    <%= stylesheet_link_tag *webpack_asset_paths('bundle', :extension => 'css'), "data-turbolinks-track" => true %>
     <%= stylesheet_link_tag 'application', "data-turbolinks-track" => true %>
     <%= yield(:stylesheets) %>
 
+    <%= stylesheet_link_tag *webpack_asset_paths('bundle', :extension => 'css'), "data-turbolinks-track" => true %>
```

Comment:
is this because we need to override something from webpack css? I'm slightly concerned with potential regressions here so I'll watch out for CSS changes

Reply:
Reverted back

Reply:
still not entirely sure why you move the line though